src/components/SignUpForm.js

```diff
@@ -76,7 +113,20 @@ export class SignUpForm extends Component {
   }
   render() {
     return (
-      <Form onSubmit={this.onSubmit}>
+      <Form onSubmit={this.onSubmit} onChange={this.loadCaptcha}>
```

Comment:
Because people mostly want to sign in and not sign up. The component mounts as soon as the "Sign In" button is clicked in the GlobalMenu. This detour prevents loading the captcha libs and assets when the user only wants to sign in.